plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -292,6 +294,22 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         return new ConnectorTableProperties();
     }
 
+    @Override
+    public void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties, TrinoPrincipal owner)
+    {
+        DatasetInfo datasetInfo = DatasetInfo.newBuilder(schemaName).build();
+        bigQueryClient.createSchema(datasetInfo);
+    }
+
+    @Override
+    public void dropSchema(ConnectorSession session, String schemaName)
+    {
+        String remoteSchemaName = bigQueryClient.toRemoteDataset(projectId, schemaName)
+                .map(RemoteDatabaseObject::getAnyRemoteName)
+                .orElse(schemaName);
```

Comment:
why `orElse`. If `bigQueryClient.toRemoteDataset` returns `Optional.empty()` it means there is not schema to be dropped. Right?

Reply:
My mistake. Changed to SchemaNotFoundException.